### PR TITLE
Refactor PlatformChannel

### PR DESCRIPTION
**Pull request by swift-kim**
- Remove `platform_channel_tizen.cc` and split its contents into separate files.
  - Move the `PlatformChannel` implementation into `platform_channel.cc`.
  - Create new files (`feedback_manager.h/cc`) for `FeedbackManager`.
  - Create new files (`tizen_shell.h/cc`) for `TizenShell` (previously `TizenWindowSystemShell`).
- Minor refactoring.
  - Remove the `sound_type` argument from `FeedbackManager.PlaySound` and create a new method `PlayTapSound`.
  - Let system UI overlay methods (`kRestoreSystemUiOverlaysMethod`, `kSetEnabledSystemUiOverlaysMethod`) always return `NotImplemented` when running on non-common profiles.
  - Remove unimplemented methods declarations (`kSetApplicationSwitcherDescriptionMethod`, `kSetEnabledSystemUIModeMethod`, `kSetSystemUIOverlayStyleMethod`).
  - Rename constants and methods (`UI`→`Ui`) for consistency.

Contributes to https://github.com/flutter-tizen/engine/issues/239. This change is a continuation of https://github.com/flutter-tizen/engine/pull/248 and https://github.com/flutter-tizen/engine/pull/249.

### CODE REVIEW

**Review by WonyoungChoi — APPROVED** [review]
LGTM

File: shell/platform/tizen/channels/platform_channel.cc [hunk 1]
```diff
@@ -130,16 +141,79 @@ void PlatformChannel::HandleMethodCall(
     }
     SetPreferredOrientations(orientations);
     result->Success();
-  } else if (method == kSetApplicationSwitcherDescriptionMethod) {
-    result->NotImplemented();
-  } else if (method == kSetEnabledSystemUIModeMethod) {
-    result->NotImplemented();
-  } else if (method == kSetSystemUIOverlayStyleMethod) {
-    result->NotImplemented();
```
Comment (bbrto21) [inline shell/platform/tizen/channels/platform_channel.cc:138]: Why don't you leave them on to-do list somewhere or comments?

Comment (swift-kim) [reply shell/platform/tizen/channels/platform_channel.cc:138]: Sure. I added these items to the existing issue (https://github.com/flutter-tizen/engine/issues/178). Thank you for your suggestion!

Comment (bbrto21) [reply shell/platform/tizen/channels/platform_channel.cc:138]: you're welcome

**Pull request merged**